data/json/recipes/recipe_ammo.json

```diff
@@ -873,6 +873,34 @@
       [ [ "lye_powder", 15 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "gelled_gasoline",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "60 m",
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "book_learn": [ [ "textbook_gaswarfare", 2 ], [ "textbook_anarch", 2, "Stuff THE MAN doesn't want you to know" ] ],
```

Comment:
> "Stuff THE MAN doesn't want you to know"

Um, what does this do?

Reply:
It hides the recipe from the book description, showing instead: "Stuff THE MAN doesn't want you to know"